### ISSUE TITLE
[gRPC] cannot specify different import path for input proto definition 

### ISSUE THREAD

**Issue by 2HgO** [ballerina-platform/ballerina-lang#25445]
**Description:**
[gRPC] code stubs for proto definitions that require multiple import path(s) (or import path(s) other than it's source directory) cannot be generated using gRPC cli tool

**Steps to reproduce:**
attempt to generate code for proto definition that imports a proto file which does not reside in its source path.

**Affected Versions:**
jBallerina 1.2.7

**OS, DB, other environment details and versions:**
MacOSX: amd64
v10.14.6

**Suggested Labels (optional):**
Type/Improvement, Area/BuildTools, Priority/Low

**Comment by daneshk** [comment]
Yes, Current implementation doesn't have a way specify give different dependent proto file directory path. the dependent proto file needs to add inside the same proto directory. For example,

if your base proto file("helloworld.proto") depends on "import foo/bar.proto" file. like below,
```proto
syntax = "proto3";

import foo/bar.proto;
```
You can add the dependent proto file("bar.proto") inside the same proto directory like
```
- helloworld.proto
  |_ foo
      |_ bar.proto
```
And then execute the same command like,
```
ballerina grpc --input helloworld.proto
```

Currently, we are looking at dependent proto files inside the same base proto file directory. We will improve the tool to provide a dependent directory path outside the base directory.

**Comment by daneshk** [comment]
Related issue #11154
